builds/installing-openshift-builds.adoc

```diff
@@ -27,7 +27,7 @@ include::modules/ob-installing-builds-using-CLI.adoc[leveloffset=+1]
 include::modules/ob-creating-a-shipwright-build-resource-cli.adoc[leveloffset=+2]
 
 
+[role="_additional-resources"]
+== Additional resources
```

Comment:
Additional resources headings need a unique id. See the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).